PROTO_tests/tests/symbol_table_test.py

```diff
@@ -0,0 +1,765 @@
+import pytest
+
+import arkouda as ak
+from arkouda.pdarrayclass import RegistrationError
+
+N = 100
+UNIQUE = N // 4
```

Comment:
These are left over and aren't referenced anywhere else

Reply:
Ah good catch. Removed.